src/app/pages/marker-generator/marker-generator.component.ts

```diff
@@ -0,0 +1,222 @@
+      {
+        // nesting columns here to allow the outer column to have width '*'
+        // while allowing the SVG to have a valid width.
+        columns: [
+          {
+            svg: beeSVG,
+            height: 40,
+            width: 40,
+            relativePosition: { x: 0, y: 0 }
+          }
+        ],
+        width: '*'
+      },
+      {
+        width: 'auto',
+        stack: [
+          p.type,
+          {
+            text: p.value,
+            fontSize: 24
+          }
+        ],
+        alignment: 'center'
+      },
+      {
+        width: '*',
+        stack: [
+          'Buzz About',
+          `Markers v${MARKERS_VERSION}`
+        ],
+        alignment: 'right'
+      }
+    ]));
+
+    const columnPages = chunk(markerColumns, numPerPage);
+
+    const backgroundSVGPages = chunk(pages.map(p => p.backgroundSVG), numPerPage);
+
+    if (numPerPage > 1 && pages.length % 2) {
+      columnPages.push([...columnPages.pop(), []]);
+      backgroundSVGPages.push([...backgroundSVGPages.pop(), blankSVG]);
+    }
+
+    const content: Content[] = columnPages.map((c, i, arr) => ({
+      columns: c.map(columns => ({
+        columns,
+        width: '*',
+        margin: 40
+      })),
+      pageBreak: arr.length - 1 === i ? null : 'after'
+    }));
+
+    const docDefinition: TDocumentDefinitions = {
+      pageSize,
+      pageOrientation,
+      content,
+      background: (currentPage, currentPageSize) => ({
+        columns: backgroundSVGPages[currentPage - 1].map(svg => ({
+          columns: [
+            {
+              svg,
+              alignment: 'center',
+              height: svgHeight,
+              relativePosition: { x: 0, y: (currentPageSize.height - svgHeight) / 2 }
+            }
+          ],
+          width: '*'
+        })),
+      }),
+      info: {
+        title: `Buzz About Markers v${MARKERS_VERSION}`,
+        author: 'Minnesota Pollinators',
+        creator: `Buzz About ${buzzAboutInfo.version} (${buzzAboutInfo.git.hash})`
+      },
+      pageMargins: 0,
+    };
+
+    return pdfMake.createPdf(docDefinition, null, fonts);
+  }
+
+  makePdf(): pdfMake.TCreatedPdf {
+
+    const pageSize: PageSize = this.markerFormGroup.controls.pageSizeControl.value;
+    const orientationAndNumPerPage: OrientationAndNumPerPage = this.markerFormGroup.controls.pageOrientationControl.value;
+    const includeFlowers: boolean = this.markerFormGroup.controls.includeFlowersControl.value;
+    const includeNests: boolean = this.markerFormGroup.controls.includeNestsControl.value;
+    const numNests: number = this.markerFormGroup.controls.numNestsControl.value;
+
+    const markerSize = 200;
+
+    return this.pdfFromPages(
+      [...(includeFlowers ? this.flowerPages(markerSize) : []), ...(includeNests ? this.nestPages(numNests, markerSize) : [])],
+      markerSize,
+      pageSize,
+      orientationAndNumPerPage
+    );
+  }
+
+  openPDF() {
+    this.makePdf().open();
+  }
+
```

Comment:
```suggestion
```
Is it okay if we get rid of this method, since it isn't used either? I know you included it for completeness, but I'm just a bit wary about having too much dead code in the project. If you really want to keep it, though, that's fine too! :)

Reply:
I'd rather keep it there as it signals that is a possible action to anyone looking in the future.